src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -0,0 +1,181 @@
+authors:
+  - petelepage
+date: 2020-02-14
+updated: 2020-02-14
+description: |
+  Use the beforeinstallprompt event to provide a custom, seamless install
+  experience for your users.
+tags:
+  - post
+draft: true
+---
+
+Many browsers make it possible for you to promote and enable the installation
+of your Progressive Web App (PWA) directly within the user interface of your
+PWA. Installation (sometimes formerly referred to as Add to Home Screen),
+makes it easy for users to install your PWA on their mobile or desktop device.
+Installing a PWA adds it to a user's launcher, allowing it to be run like any
+other installed app.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="spotify-custom-install.png"
+       alt="Install App button provided in the Spotify PWA">
+  <figcaption class="w-figcaption">
+    "Install App" button provided in the Spotify PWA
+  </figcaption>
+</figure>
+
+Why would you want a user to install your app? The same reason you'd want a
+user to install your app from any app store. Users who install are your most
+engaged audience. Users who install a PWA have better engagement metrics than
+typical visitors, including more repeat visits, longer time on site and
+higher conversion rates, often at parity with native app users on mobile
+devices.
+
+<div class="w-clearfix"></div>
+
+## Promoting installation
+
+To indicate your Progressive Web App is installable, and to provide a custom
+in-app install flow:
+
+1. Listen for the `beforeinstallprompt` event.
+2. Save the `beforeinstallprompt` event, so it can be used to trigger the
+   install flow later.
+3. Alert the user your PWA is installable, and provide a button or other
+   element to start the in-app installation flow.
+
+### Listen for the beforeinstallprompt event
+
+If your Progressive Web App meets the required [installation criteria](/install-criteria/),
+the browser fires a `beforeinstallprompt` event. Save a reference to the
+event, and update your user interface to indicate that the user can install
+your PWA. This is highlighted below.
+
+```js/5-8
+let deferredPrompt;
+
+window.addEventListener('beforeinstallprompt', (e) => {
+  // Prevent the mini-infobar from appearing on mobile
+  e.preventDefault();
+  // Stash the event so it can be triggered later.
+  deferredPrompt = e;
+  // Update UI notify the user they can install the PWA
+  showInstallPromotion();
+});
+```
+
+There are many different [patterns](/install-patterns/) that you can use to
+notify the user your app can be installed and provide an in-app install
+flow, for example, a button in the header, an item in the navigation menu,
+or an item in your content feed.
+
+{% Aside 'caution' %}
+You may want to wait before showing the prompt to the user so you don't
+distract them from what they're doing. For example, if the user is in a
+check-out flow, or creating their account, let them complete that before
+interrupting them with the prompt.
```

Comment:
'interrupting them with the prompt' --> 'prompting them'.

Pithier writing isn't the only issue. I think we should avoid implying that interrupting is good app behavior.

Reply:
Good point - fixed.